.gitignore

```diff
@@ -16,3 +16,11 @@ vendor/**/composer.lock
 /vue/dist/*.common.js
 /vue/dist/*.map
 /vue/dist/*.development.*
+vendor/bin/phpcs
+vendor/bin/phpcbf
+vendor/dealerdirect
+vendor/matomo-org/matomo-coding-standards
+vendor/phpstan
+vendor/slevomat
+vendor/squizlabs
+phpcs-report.xml
```

Comment:
Do we need to ignore these? We're not committing anything so dirty git status is not an issue, is it?

Reply:
Its not needed anymore :+1: 
Removed